docs/how-to-define-a-language.md

```diff
@@ -0,0 +1,7 @@
+# How Languages Are Defined
+
+1. Add the new language code to the array `supportedLanguages` in [../src/i18n/baseConfig.ts](/src/i18n/baseConfig.ts)
+2. Add an entry to default language file [../public/locales/en-US/translations.json]() with the key being `language_{languageCode}`. ex. `language_en-US`
+3. Create a folder in [../public/locales/](/public/locales) with the language code as its name.
+4. Add a new file, `translations.json`, to the new folder
+5. Translate all the entries.  If you prefer to use the english version just set the value to `null`.
```

Comment:
Can you link to an example / template file? (Or just include a code snippet which shows how to set up the entries?)

Just from reading this I'm not clear on how to format them. Is there a file I'm supposed to be copying from to learn what each entry is named?

Reply:
I can add a link to the Japanese file.